docs/installation/installation-using-docker-windows-10.md

```diff
@@ -43,25 +43,27 @@ We have prepared installation script for Debian dependencies in order to speed u
 In `Debian` application run these commands:
 * `sudo apt update`
 * `sudo apt install -y --no-install-recommends wget`
-* `wget --no-check-certificate https://raw.githubusercontent.com/shopsys/shopsys/tl-update-windows-installation-guide/project-base/scripts/install-docker-wsl-debian.sh`
+* `wget --no-check-certificate https://raw.githubusercontent.com/shopsys/shopsys/master/project-base/scripts/install-docker-wsl-debian.sh`
 * `sudo bash install-docker-wsl-debian.sh`
 * You can delete the downloaded file at the end of installation using `rm install-docker-wsl-debian.sh` command.
 * To ensure everything is working alright, restart `Docker Desktop` and `Debian` applications.
 
 !!! note
-    If you prefer to know more about what is happening during installation, open [install-docker-wsl-debian.sh](https://github.com/shopsys/shopsys/tl-update-windows-installation-guide/project-base/scripts/install-docker-wsl-debian.sh) script and run all commands manually.
+    If you prefer to know more about what is happening during installation, open [install-docker-wsl-debian.sh](https://github.com/shopsys/shopsys/master/project-base/scripts/install-docker-wsl-debian.sh) script and run all commands manually.
```

Comment:
I have just notices the path is missing `/tree` in it - e.g. `https://github.com/shopsys/shopsys/tree/master/docker/conf`

```suggestion
    If you prefer to know more about what is happening during installation, open [install-docker-wsl-debian.sh](https://github.com/shopsys/shopsys/tree/master/project-base/scripts/install-docker-wsl-debian.sh) script and run all commands manually.
```